spec.bs

```diff
@@ -309,13 +309,13 @@ console.assert(producedValues.length === 0);
 callback SubscribeCallback = undefined (Subscriber subscriber);
 callback ObserverCallback = undefined (any value);
 
-dictionary Observer {
+dictionary SubscriptionObserver {
   ObserverCallback next;
   ObserverCallback error;
   VoidFunction complete;
 };
 
-typedef (ObserverCallback or Observer) ObserverUnion;
+typedef (ObserverCallback or SubscriptionObserver) ObserverUnion;
```

Comment:
I wonder if we should rename `ObserverCallback` to `SubscriptionObserver` callback, in the spirit of specificity and to keep these consistent?

Reply:
I've done that, let me know what you think.